Reconstruction/RecCalorimeter/src/components/CreatePositionedHit.cpp

```diff
@@ -0,0 +1,121 @@
+#include "CreatePositionedHit.h"
```

Comment:
In fact this will not be needed with the new version of the EDM. I think I will remove this and add the position information in the main CreateCaloCells algorithm once we have the "PositionHits". I might remove it if you wish.